deps/wazuh_testing/setup.py

```diff
@@ -55,12 +56,18 @@ def get_files_from_directory(directory):
             paths.append(os.path.join('..', path, filename))
     return paths
 
+def get_version():
+    f = open('../../version.json')
+    data = json.load(f)
+    version = data['version']
+    return version
+
 
 package_data_list.extend(get_files_from_directory('wazuh_testing/qa_docs/search_ui'))
 
 setup(
     name='wazuh_testing',
-    version='4.4.0',
+    version=get_version(),
```

Comment:
Try to install the framework from different paths to make sure there are no problems when launching pipelines (perhaps from one playbook to another may vary the way it is installed). It is a simple test and we save us a headache, I suggest these tests:

- Install it from the repository root
- Install it from `deps/`
- Install it from `deps/wazuh_testing`
- Install from outside the repository (ex: repo in `/wazuh-qa/` install from `/`)

Reply:
In fact, I wouldn't say I like to use relative paths in the `open()` built-in function because it can lead to errors while searching for the file. So, it is better, IMHO, to use Pathlib to handle this.

Reply:
It has been changed in https://github.com/wazuh/wazuh-qa/pull/4172/commits/8d4a1f1bf6d3d4705e5ea2ead93b25d30dac3874 to use a relative path, but specifying it is from the script's location. All proposed tests have been done and it works correctly.